sdk/search/Azure.Search.Documents/tests/SearchIndexerClientTests.cs

```diff
@@ -381,7 +381,7 @@ public async Task CrudSkillset()
         {
             await using SearchResources resources = await SearchResources.CreateWithBlobStorageAndIndexAsync(this);
 
-            SearchIndexerClient client = resources.GetIndexerClient();
+            SearchIndexerClient client = resources.GetIndexerClient(new SearchClientOptions());
```

Comment:
Does this need the service version?

Reply:
It defaults to latest.

Reply:
Remember those bugs where recordings didn't match when you updated the service version?  I think these will cause more of those if you don't pass the version through.